app/scripts/directives/display-event-data/birth.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<table>
+  <tr>
+    <td>Baby Given Name:</td>
+    <td>{{event.data.babyGivenName}}</td>
+  </tr>
+  <tr>
+    <td>Baby Middle Names:</td>
+    <td>{{event.data.babyMiddleNames}}</td>
+  </tr>
+  <tr>
+    <td>Baby Family Name:</td>
+    <td>{{event.data.babyFamilyName}}</td>
+  </tr>
+  <tr>
+    <td>Place of Birth:</td>
+    <td>{{event.data.birthPlace}}</td>
+  </tr>
+  <tr>
+    <td>Birth Date:</td>
+    <td>{{event.data.birthDate | date:'yyyy-MM-dd'}}</td>
+  </tr>
+  <tr>
+    <td>PMothers Given Name:</td>
```

Comment:
Maybe we should add the possessive apostrophe to all the pronouns.  Mother's and Babay's